airflow/providers/celery/provider.yaml

```diff
@@ -22,12 +22,13 @@ description: |
     `Celery <http://www.celeryproject.org/>`__
 
 versions:
+  - 2.1.0
   - 2.0.0
   - 1.0.1
   - 1.0.0
 
 additional-dependencies:
-  - apache-airflow>=2.1.0
+  - apache-airflow>=2.2.0
```

Comment:
Shouldn't this depend on celery directly too? If one didn't use the `celery` extra to install the provider, instead installing `apache-airflow-providers-celery`, you could end up without celery, right?

Reply:
Nevermind, looks like there is some magic that gets the celery deps into the provider from setup.py.

Reply:
Yep - you are totally right! 

It can also  help to solve the problem of dependencies with dev version we have. Plus - when we combine it with #18020  we should have all the protection against any conflicting versions in place.

Reply:
Ahhh. Works now :)